sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2657,22 +2661,27 @@ object functions {
 
   /**
    * Converts time string in format yyyy-MM-dd HH:mm:ss to Unix timestamp (in seconds),
-   * using the default timezone and the default locale, return null if fail.
+   * using the default timezone and the default locale.
+   * Returns `null` if fails.
+   *
    * @group datetime_funcs
    * @since 1.5.0
    */
   def unix_timestamp(s: Column): Column = withExpr {
     UnixTimestamp(s.expr, Literal("yyyy-MM-dd HH:mm:ss"))
   }
 
+  // scalastyle:off line.size.limit
   /**
-   * Convert time string with given pattern
-   * (see [http://docs.oracle.com/javase/tutorial/i18n/format/simpleDateFormat.html])
-   * to Unix time stamp (in seconds), return null if fail.
+   * Converts time string with given pattern to Unix timestamp (in seconds).
+   * Returns `null` if fails.
+   *
+   * @see <a href="http://docs.oracle.com/javase/tutorial/i18n/format/simpleDateFormat.html">Customizing Formats</a>
```

Comment:
```
@see <a href="http://docs.oracle.com/javase/tutorial/i18n/format/simpleDateFormat.html">
Customizing Formats</a>
```

Given my past try, the above one works - https://github.com/apache/spark/pull/15999#discussion_r89456806 (IIRC, I checked the output).